components/lib/crash/metrics.yaml

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file defines the metrics that are recorded by glean telemetry. They are
+# automatically converted to Kotlin code at build time using the `glean_parser`
+# PyPI package.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+crash_metrics:
```

Comment:
Do we want these metrics to be in the `metrics` ping?

Reply:
Good question, I assume this would be a good thing to have in `baseline` or `glean_client_info`?

Reply:
I'm not sure about that. `baseline` is for things common to all products. Not all products might use `lib-crash`, so it feels this should go in `metrics`. @georgf , thoughts?

Reply:
This is a measure owned by Glean, as such it should go into its default ping - the `metrics` ping - unless there are good arguments otherwise.

Reply:
Sounds good to me, I think that makes sense.